ethereum/extethclient/eth_wallet_client.go

```diff
@@ -162,7 +162,7 @@ func (c *ethClient) SetGasPrice(gasPrice uint64) {
 		c.gasPrice = nil
 		return
 	}
-	c.gasPrice = big.NewInt(0).SetUint64(gasPrice)
+	c.gasPrice = new(big.Int).SetUint64(gasPrice)
```

Comment:
I just did a global replace on `big.NewInt(0).` to `new(big.Int).`. If it didn't have an extra dot at the end, it didn't get replaced, as those are instances where the user needed a zero value `big.Int` verses an arbitrary `big.Int` object. (Although the standard library uses `new(big.Int)` for both.)